packages/markdown-pdf/src/PdfTransformer.js

```diff
@@ -173,10 +171,40 @@ class PdfTransformer {
      * @param {*} outputStream - the output stream
      */
     async toPdf(input, options, outputStream) {
-
         const printer = new PdfPrinter(fonts);
 
-        if(!input.getType) {
+        // Asynchronously retrieve image as Base64 encoded data from URL.
+        const getRemoteImageData = async (url) => {
+            try {
+                const buffer = await axios.get(url, { responseType: 'arraybuffer' });
+                return ("data:image/png;base64," + buffer.data.toString('base64'));
```

Comment:
How do you know that the buffer returned in a PNG?

Reply:
Ummm... I don't. It was just that the raw `buffer.data` was large and thought maybe using base64 png would be nicer.  Could maybe make this an option too?

Reply:
Ok... Am changing logic to use `Buffer.from(buffer.data)` with no base64 encoding.